### DISCUSSION TITLE
Welcome to HyperDbg Discussions!

### DISCUSSION THREAD

**Discussion by SinaKarvandi** [HyperDbg/HyperDbg#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hsiegersma** [comment]
Hi uhh, im quite new to all of this but im about to start working on AMD processor support for HyperDbg and i wanted to let you know i am about to do so 8)

**Reply by SinaKarvandi** [reply]
Hey,

That would be great. Looking forward to seeing how it will work, and also please merge it to HyperDbg once it's done.
